### PR TITLE
Add SNI support in release notes for 0.8

### PR DESCRIPTION
**Pull request by andraxylia**

### CODE REVIEW

File: content/about/notes/0.8.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ increases effective scalability and should eliminate spurious 404 errors. [TBD:
 - **Gateway for Ingress/Egress**. We no longer support combining Kubernetes Ingress specs with Istio route rules,
 as it has led to several bugs and reliability issues. Istio now supports a platform independent ingress/egress
 [Gateway](/docs/concepts/traffic-management/rules-configuration/#gateways) that works across
-Kubernetes and Cloud Foundry and works seamlessly with the routing rules.
+Kubernetes and Cloud Foundry and works seamlessly with the routing rules. The Istio Gateway supports SNI [Server Name Indication]
```
Comment (geeknoid) [inline content/about/notes/0.8.md]: Cloud Found4y both work seamlessly

Comment (andraxylia) [reply content/about/notes/0.8.md]: It's the ingress/egress that works.

Comment (geeknoid) [reply content/about/notes/0.8.md]: The sentence is not right:

"k8s and cloud foundry and works"

Comment (linsun) [reply content/about/notes/0.8.md]: Do we have a link to refer to users for use SNI in istio?

Comment (andraxylia) [reply content/about/notes/0.8.md]: Not yet, we can update later.

Comment (rshriram) [reply content/about/notes/0.8.md]: There is nothing to do with SNI. 
I would call out SNI separately. Simply state that HTTPS access to external services will be auto configured based on SNI. 
Also state that Gateways support SNI based routing as well as serving certificate based on the SNI value.

File: content/about/notes/0.8.md [hunk 2]
```diff
@@ -14,14 +14,17 @@ This model adds many new features and addresses usability issues
...
 - **Gateway for Ingress/Egress**. We no longer support combining Kubernetes Ingress specs with Istio route rules,
 as it has led to several bugs and reliability issues. Istio now supports a platform independent ingress/egress
 [Gateway](/docs/concepts/traffic-management/rules-configuration/#gateways) that works across
 Kubernetes and Cloud Foundry and works seamlessly with the routing rules.
 
+- **Gateways support [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication)** based routing
+as well as serving a certificate based on the SNI value. HTTPS access to external services will be auto configured based on SNI.
+
+- **Envoy V2**. Users can choose to inject Envoy V2 as the sidecar. In this mode, Pilot uses Envoy's new API to push configuration to the data plane. This new approach
+increases effective scalability and should eliminate spurious 404 errors. [TBD: docs on how to control this?]
```
Comment (geeknoid) [inline content/about/notes/0.8.md]: Let's remove the TBD entry. This page will be going live RSN.

Comment (andraxylia) [reply content/about/notes/0.8.md]: Not sure what you mean, there is nothing TBD. SNI is a big feature in 0.8., please approve asap.

Comment (linsun) [reply content/about/notes/0.8.md]: I think Martin meant "[TBD: docs on how to control this?]"

Comment (andraxylia) [reply content/about/notes/0.8.md]: Oh, I see, I did not add this, but let me remove it.

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 138855ff7bbd293f7e04fd30d12e6662260602b5

https://app.netlify.com/sites/preliminary-istio/deploys/5b10172cdd6a547833edd5a3

**Comment by linsun** [comment]
@andraxylia can you fix the deploy error?

File: content/about/notes/0.8.md [hunk 3]
```diff
@@ -14,13 +14,17 @@ This model adds many new features and addresses usability issues
...
 - **Gateway for Ingress/Egress**. We no longer support combining Kubernetes Ingress specs with Istio route rules,
 as it has led to several bugs and reliability issues. Istio now supports a platform independent ingress/egress
 [Gateway](/docs/concepts/traffic-management/rules-configuration/#gateways) that works across
-Kubernetes and Cloud Foundry and works seamlessly with the routing rules.
+Kubernetes and Cloud Foundry and integrates seamlessly with the routing rules.
+
+- **The new Gateway supports [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication)** based routing,
+as well as serving a certificate based on the SNI value. HTTPS access to external services will be auto configured based on SNI.
+
+- **Envoy V2**. Users can choose to inject Envoy V2 as the sidecar. In this mode, Pilot uses [Envoy's v2 Aggregated Discovery Service
```
Comment (ZackButcher) [inline content/about/notes/0.8.md]: I think the phrase "Envoy V2 as the sidecar" gives the wrong impression (that it's a new Envoy version; Envoy has only released up to a version 1.6). I think

> **Envoy v2 APIs**. Users can choose to inject a sidecar using Envoy's v2 APIs. In this mode, Pilot uses ...

works a bit better.

Sorry, I know this diff is just from moving the text around and you didn't write this originally, but I think we should clean it up while we're here.

Comment (andraxylia) [reply content/about/notes/0.8.md]: Np, done

**Comment by andraxylia** [comment]
@linsun done, please approve

**Pull request merged**